modules/standard/Random.chpl

```diff
@@ -1053,6 +1054,7 @@ module Random {
     }
 
 
+    module PCGRandomLib {
```

Comment:
The later commit moves it out to a sister module to PCGRandom, which I think is better code organization anyway.